### PR TITLE
[PS-5543] Increase prefetch count for tournament conclude service and leaderboard service

### PR DESCRIPTION
**Pull request by david-thornton717**
Set the Prefetch count for the tournament conclude service to five when we create the bean

@RuoanJi 
@vidurchanana 
@zgarbowitz

### CODE REVIEW

**Comment by RuoanJi** [comment]
can you also set the same number for `LeaderboardService`?

Long shot but It might solve the leaderboard conclude issue.

File: src/groovy/org/grails/rabbitmq/RabbitServiceConfigurer.groovy [hunk 1]
```diff
@@ -216,6 +217,10 @@ class RabbitServiceConfigurer {
             connectionFactory = ref("rabbitMQConnectionFactory")
             concurrentConsumers = serviceConcurrentConsumers
             messageListener = ref("${propertyName}RabbitAdapter")
+
+            if (type != SUBSCRIBE && serviceGrailsClass.clazz.name == 'skillz.TournamentConcludeService') {
```
Comment (zgarbowitz) [inline src/groovy/org/grails/rabbitmq/RabbitServiceConfigurer.groovy]: Why do we care about type? Do you know what this value is generally set to?

Comment (david-thornton717) [reply src/groovy/org/grails/rabbitmq/RabbitServiceConfigurer.groovy]: It is normally set to `rabbitQueue`

**Comment by zgarbowitz** [comment]
👍

**Comment by vidurchanana** [comment]
Besides the PR name to include the Leaderboard service, rest looks good 👍

**Comment by RuoanJi** [comment]
👍

**Pull request merged**